azure-mgmt-resources/src/main/java/com/microsoft/azure/management/resources/fluentcore/collection/ResourcesInGroup.java

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.management.resources.fluentcore.collection;
+
+public interface ResourcesInGroup<T, DefinitionT> extends
+    SupportsListing<T>,
+    SupportsCreating<DefinitionT>,
+    SupportsDeleting {}
```

Comment:
strictly speaking, .getById(id) may not be needed in the simple resources we have modeled so far, as long as .resourceGroupName() is available for every groupable resource, which we did make sure it is. 
But I think once we dig deeper into Networking and similarly complex areas, the need for .get(id) may become more obvious -- remember you can associate resources with each other from different resource groups.
For now, i'm ok erring on the less-is-more side and removing .get(id). But I suspect it will come back later :-)